ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-release-4.18.yaml

```diff
@@ -87,6 +87,24 @@ tests:
           cpu: 100m
     - ref: openshift-e2e-test
     workflow: openshift-e2e-aws
+- as: openshift-e2e-aws-techpreview
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*\/)?(?:\.gitignore|DOWNSTREAM_OWNERS|DOWNSTREAM_OWNERS_ALIASES|LICENSE|DCO|CODEOWNERS)|\.golangci\.yaml|\.goreleaser\.yml|\.markdownlint.yaml|^\.github|Tiltfile$
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: upstream-e2e-techpreview
+      commands: make test-e2e
+      dependencies:
+      - env: TEST_CATALOG_IMAGE
+        name: test-catalog
+      from: src
+      resources:
+        requests:
+          cpu: 100m
```

Comment:
I'm a little leery of _also_ running the upstream e2e in techpreview.

Reply:
I admit I was surprised to see it as well, but perhaps it's just a naming thing, as this test is currently run in [4.17](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-release-4.17.yaml#L82-L99) the same way.